src/rust/engine/src/externs/mod.rs

```diff
@@ -183,7 +176,7 @@ pub fn hasattr(value: &Value, field: &str) -> bool {
 ///
 /// Gets an attribute of the given value as the given type.
 ///
-fn getattr<T>(value: &Value, field: &str) -> Result<T, String>
+fn getattr<T>(value: &PyObject, field: &str) -> Result<T, String>
```

Comment:
I think this is a fine name to keep using for the parameter, but consider renaming to object or py_object